### ISSUE TITLE
doccumentation issue: ipc subscribe to iot core c++ example doesn't compile

### ISSUE THREAD

**Issue by clogwog** [awsdocs/aws-iot-greengrass-v2-developer-guide#17]
The C++ example at https://github.com/awsdocs/aws-iot-greengrass-v2-developer-guide/blob/main/doc_source/interprocess-communication.md#ipc-operation-subscribetotopic-example-cpp at line 767 does not compile:

```
mkdir -p build
g++ -o build/main.o src/main.cpp -c -std=c++11 -g  -pthread -Isrc -I/usr/include/glib-2.0 -I/usr/lib/aarch64-linux-gnu/glib-2.0/include -I/usr/include/aarch64-linux-gnu -I/usr/src/jetson_multimedia_api/include -I/opt/awsiotsdk/include
src/main.cpp: In function ‘int main()’:
src/main.cpp:80:10: error: ‘response’ was not declared in this scope
     if (!response) {
          ^~~~~~~~
src/main.cpp:80:10: note: suggested alternative: ‘remove’
     if (!response) {
          ^~~~~~~~
          remove
Makefile:48: recipe for target 'build/main.o' failed
make: *** [build/main.o] Error 1

```
I'd like to put in  pull request, but i'm actually having trouble with other parts of this example as well, so I'm the least likely to help with a pull request at the moment.

**Comment by clogwog** [comment]
i think the pull request should do it.

**Comment by FThompsonAWS** [comment]
Thanks, I've merged the PR #18 to fix this issue. I've updated the documentation website with this fix and a fix for the C++ PublishToTopic example, which had the same line missing.
